frontend/src/layout/UserMenu.js

```diff
@@ -18,6 +18,10 @@ const MyAddressMenu = forwardRef(({ onClick, label }, ref) => (
   <MenuItemLink ref={ref} to="/Location" primaryText={label} leftIcon={<HomeIcon />} onClick={onClick} />
 ));
 
+const AccountSettingsMenu = forwardRef(({ onClick, label, profileUri }, ref) => (
+  <MenuItemLink ref={ref} to={'/account/settings'} primaryText={label} leftIcon={<HomeIcon />} onClick={onClick} />
```

Comment:
The `profileUri` param is not used/needed.
Also, please use SettingsIcon instead of HomeIcon :-)
`import SettingsIcon from '@material-ui/icons/Settings';`
https://v4.mui.com/components/material-icons/

Reply:
It should be added now. ^^